content/tutorials/rust/mysql-cdc.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Convert MySQL table changes into events streams (Rust)
+toc: true
+---
+
+In this tutorial, we're going to be talking about **Change Data Capture** (or CDC).
+We'll give a brief overview of what CDC is, then we'll talk about the component
+steps in a CDC setup. Then, we'll take a look at how we can implement our own
+CDC system using Fluvio.
+
+Roughly speaking, CDC is a way to record updates to a database in a structured way,
+so that those updates may be replayed on another database instance to create a
+perfect copy. We typically refer to the database being copied as the "leader", and
+any database which is being replicated from it as a "follower". Note that there can
+be only one leader, but potentially many followers.
+
+In this CDC tutorial, we'll be using a [Rust MySQL Binlog] library to watch for
+updates to our leader database. When we see an update, we'll send it as a message
+on a Fluvio stream. We'll call the program that does this the "Producer", since
+it "produces" messages to the Fluvio topic. We'll also create a second program
+that watches for messages to appear on our Fluvio topic and then write those
+messages to the binlog of the follower database. This program will be called the
+"Consumer", since it "consumes" messages from Fluvio.
+
```

Comment:
Node, "Producer" convert all MySqL operations into durable events that maybe utilized for a variety of other use cases: analytics, audit trails, notifications, etc. (or something to that effect)